config.microsoftInternal.dsc

```diff
@@ -27,22 +27,22 @@ export const pkgs = isMicrosoftInternal ? [
     { id: "WindowsSdk.Corext", version: "10.0.16299.1", alias: "Windows.Sdk" },
 
     // Artifact packages and dependencies
-    { id: "Microsoft.VisualStudio.Services.ArtifactServices.Shared", version: "17.149.28711-buildid8812231", dependentPackageIdsToSkip: ["*"], dependentPackageIdsToIgnore: ["BuildXL.Cache.Hashing"] },
-    { id: "Microsoft.VisualStudio.Services.BlobStore.Client", version: "17.149.28711-buildid8812231", dependentPackageIdsToSkip: ["*"], dependentPackageIdsToIgnore: ["BuildXL.Cache.Hashing", "BuildXL.Cache.Interfaces"] },
-    { id: "Microsoft.VisualStudio.Services.Client", version: "17.149.20190311.3-release", dependentPackageIdsToSkip: [ "Microsoft.Net.Http", "Microsoft.AspNet.WebApi.Client", "System.Security.Cryptography.OpenSsl", "System.Security.Principal.Windows" ] },
-    { id: "Microsoft.VisualStudio.Services.InteractiveClient", version: "17.149.20190311.3-release", dependentPackageIdsToSkip: [ "Ben.Demystifier" ], dependentPackageIdsToIgnore: [ "Ben.Demystifier" ] },
+    { id: "Microsoft.VisualStudio.Services.ArtifactServices.Shared", version: "17.150.28901-buildid9382555", dependentPackageIdsToSkip: ["*"], dependentPackageIdsToIgnore: ["BuildXL.Cache.Hashing"] },

```

Comment:
>17.150.28901-buildid9382555 [](start = 79, length = 27)

Since you are updating this, perhaps you can declare a `const` at the top of the file with the artifacts version?

Reply:
Sorry, but that would require another full build. I'm opting for a quicker release instead.